src/lightning_app/core/work.py

```diff
@@ -537,6 +537,14 @@ def _cleanup_calls(calls: Dict[str, Any]):
                         final_statuses.append(status)
                 calls[call_hash]["statuses"] = final_statuses
 
+    def start(self):
```

Comment:
qq: this is call blocking?

for eg:
```py
class Someflow(LightningFlow):
    def run(self):
        self.my_work.start(). <- is this blocking in the sense will the program move forward or wait for provisioning?
        print('something') 
```

Reply:
This depends if your work is running with `parallel=True` or not. But default, it is blocking.